File: src/data/Model/Cms360/Seating Plan/DateSheetDetail.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model

{
    [Table("DateSheetDetail", Schema = "ExamSeatingPlan")]

    public class DateSheetDetail
    {
        [Key]
        public Guid DateSheetDetailId { get; set; }
        public DateTime DateSheetDate { get; set; }
        public TimeSpan FromTime { get; set; }
        public TimeSpan ToTime { get; set; }
        public Guid DateSheetId { get; set; }
        public int StatusId { get; set; }

    }
    [Table("VWDateSheetDetail", Schema = "ExamSeatingPlan")]
    public class VWDateSheetDetail
    {
        [Key]
        public Guid DateSheetDetailId { get; set; }
        public Guid DateSheetId { get; set; }
        public Guid CampusProgramId { get; set; }
        public string ExamName { get; set; }
        public string CampusName { get; set; }
        public string Description { get; set; }
        public string BuildingName { get; set; }
        public string RoomName { get; set; }
        public Guid RoomBuildingLinkId { get; set; }
        public int StatusId { get; set; }
        public Guid CampusId { get; set; }
        public Guid SessionId { get; set; }
        public Guid ProgramDetailId { get; set; }
        public DateTime DateSheetDate { get; set; }
        public TimeSpan FromTime { get; set; }
        public TimeSpan ToTime { get; set; }
    }
}

File: src/ui/app/root/application/models/Seating Plan/datesheetdetail.ts
import { Time } from "highcharts";


export interface ISeatingPlanDateSheetDetail {
    dateSheetDetailId: string;
    dateSheetDate: Date;
    fromTime: string;
    toTime: string;
    dateSheetId: string;
    statusId: number;
}


export interface IVWDateSheetDetail {
    dateSheetDetailId: string;
    dateSheetId: string;
    campusProgramId: string;
    examName: string;
    campusName: string;
    description: string;
    buildingName: string;
    roomName: string;
    roomBuildingLinkId: string;
    statusId: number;
    campusId: string;
    sessionId: string;
    programDetailId: string;
    dateSheetDate: Date;
    fromTime: Time;
    toTime: Time;
}